css/templates/e-commerce/page.css

```diff
@@ -328,6 +332,14 @@ amp-selector [option][selected] {
   text-indent: -var(--space-2);
 }
 
+.ampstart-headerbar-fixed-link-wrapper {
+  width: 25px;
+}
+
+.ampstart-headerbar-fixed-link {
+  margin-right: 0;
```

Comment:
I couldn't find an elegant way of doing this without changing all pages that include the nav because mr2 is already hardcoded into the template.